index.html

```diff
@@ -122,13 +124,62 @@ <h5><a target="_blank" href="https://github.com/pcengines/apu2-documentation/blo
 
         <div class="container">
             <!-- mainline releases v4.8.0.2 -->
-<!-- 
-    <div class="break"></div> 
-    insert this code when a new update comes out to separate a new version from the current.
--->
+ 
+<!--<div class="break"></div>insert this code when a new update comes out to separate a new version from the current.-->
 
-        <div class="smaller-margin" id="mr-12">
+<div class="smaller-margin" id="mr-13">
             <h2 class="text-middle">Mainline releases</h2>
+            <!-- h2 tag needs to be moved to the newest version -->
+            <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.8.0.2...v4.8.0.3">v4.8.0.3</a></h4>
+        </div>
+
+        <div class="dimmed-text"><p>Release date: 2018-08-07</p></div>
+
+        <ul>
+            <li>Fixed/added:</li>
+            <ol>
+                <li>synced and rebased with coreboot official repo commit c8e974f</li>
+                <li>change of date format in sign of life string now applies only to PC Engines</li>
+                <li>use CONFIG_LOCALVERSION as a coreboot version string</li>
+                <li>double sign-of-life during power on</li>
+                <li>Enabled IOMMU</li>
```

Comment:
@mscibisz everywhere small letters on list and here the only one capital letter. Align to the small letter format.